library/std/src/sys/theseus/os.rs

```diff
@@ -1,25 +1,46 @@
-use super::unsupported;
-use crate::error::Error as StdError;
-use crate::ffi::{OsStr, OsString};
-use crate::fmt;
-use crate::io;
-use crate::marker::PhantomData;
-use crate::path::{self, PathBuf};
+use super::{current_task, current_task_id, io_err, unsupported};
+use crate::{
+    error::Error as StdError,
+    ffi::{OsStr, OsString},
+    fmt, io,
+    marker::PhantomData,
+    path::{self, PathBuf},
+};
 
 pub fn errno() -> i32 {
-    0
+    panic!("should not be used on this target");
```

Comment:
this is good for now since it'll reveal all the places where `errno()` calls are actually run. However, isn't `errno()` somewhat "required" by various parts of `std:error` and/or `std::io` error stuff?

Reply:
AFAIK it's only used in `io::Error::last_os_error` which is a public `std` function not used internally. I think it makes sense to panic if an `std` user tries to call it.

Reply:
fine by me, especially for an initial version.